pkg/webui/oauth/views/change-password/index.js

```diff
@@ -0,0 +1,161 @@
+// Copyright © 2019 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import React from 'react'
+import bind from 'autobind-decorator'
+import { defineMessages } from 'react-intl'
+import { push, replace } from 'connected-react-router'
+import { connect } from 'react-redux'
+import * as Yup from 'yup'
+
+import api from '../../api'
+import sharedMessages from '../../../lib/shared-messages'
+import { digit, uppercase, special } from '../../lib/regexp.js'
+
+import Button from '../../../components/button'
+import Form from '../../../components/form'
+import Input from '../../../components/input'
+import SubmitButton from '../../../components/submit-button'
+import Logo from '../../../components/logo'
+import IntlHelmet from '../../../lib/components/intl-helmet'
+import Message from '../../../lib/components/message'
+
+import style from '../login/login.styl'
+
+const m = defineMessages({
+  changePassword: 'Change Password',
+})
+
+const validationSchema = Yup.object().shape({
+  new_password: Yup.string()
+    .min(8)
+    .matches(digit, sharedMessages.validatePasswordDigit)
+    .matches(uppercase, sharedMessages.validatePasswordUppercase)
+    .matches(special, sharedMessages.validatePasswordSpecial)
+    .required(sharedMessages.validateRequired),
+  new_confirm: Yup.string()
+    .oneOf([ Yup.ref('new_password'), null ], sharedMessages.validatePasswordMatch)
+    .min(8)
+    .required(sharedMessages.validateRequired),
+})
+
+const initialValues = {
+  new_password: '',
+  new_confirm: '',
+}
+
+@connect(null, {
+  push,
+  replace,
+})
+@bind
+export default class ChangePassword extends React.PureComponent {
+  constructor (props) {
+    super(props)
+    this.state = {
+        error: '',
+        info: '',
+    }
+  }
+
+  async handleSubmit (values, { setSubmitting }) {
+    const { match: {params }, push } = this.props
```

Comment:
```suggestion
    const { match: { params }, push } = this.props
```

As @bafonins said, check your editor setup.